test/test_operations_constructor.jl

```diff
@@ -102,4 +105,58 @@ end
         @test ED.optimizer_log[:primal_status] == MOI.FEASIBLE_POINT
         @test UC.optimizer_log[:primal_status] == MOI.FEASIBLE_POINT
     end
+
+    @testset "AC branch Branch rate constraints" begin
+        thermal_model = DeviceModel(ThermalStandard, ThermalDispatch)
+        devices = Dict{Symbol, DeviceModel}(
+            :Generators => DeviceModel(ThermalStandard, ThermalDispatch),
+            :Loads => DeviceModel(PowerLoad, StaticPowerLoad),
+        )
+        branches = Dict{Symbol, DeviceModel}(
+            :L => DeviceModel(PSY.MonitoredLine, PSI.FlowMonitoredLine),
+        )
+        template = OperationsProblemTemplate(ACPPowerModel, devices, branches, services)
+        system = c_sys5
+        line = PSY.get_component(Line, system, "1")
+        PSY.convert_component!(MonitoredLine, line, system; force = true)
```

Comment:
Why are you forcing here?

Reply:
it doesn't need the force = true, I took it out